src/controller/CHIPDevice.cpp

```diff
@@ -681,6 +681,19 @@ CHIP_ERROR Device::SendReadAttributeRequest(app::AttributePathParams aPath, Call
     return err;
 }
 
+Messaging::ExchangeContext * Device::GetNewExchangeContext(Messaging::ExchangeDelegate * delegate)
+{
+    if (mExchangeMgr == nullptr || delegate == nullptr)
+    {
+        return nullptr;
+    }
+
+    bool didLoad = false;
```

Comment:
Why not call the no-arg version of `LoadSecureSessionParametersIfNeeded` if we don't care about whether we loaded?

Reply:
Didn't even realize this was an option. Will change this if we decide to keep this vs. exposing SecureSession